src/hevm/README.md

```diff
@@ -58,20 +58,37 @@ set the `TERMINFO` environment variable; on Ubuntu, you should do
 
 ### Building with Stack or Cabal
 
-If you can't or won't use Nix, the easiest way especially if you don't
-have GHC (the Haskell compiler) installed already, is to use
-[Stack](https://docs.haskellstack.org/en/stable/README/), which can
-take care of installing GHC for you.  These commands should work:
-
-    $ curl -sSL https://get.haskellstack.org/ | sh
-    $ git clone https://github.com/dapphub/hevm
-    $ cd hevm && stack setup && stack install
-
-If you prefer to use your own installation of GHC and the basic
-Haskell package manager, Cabal, simply run:
-
-    $ git clone https://github.com/dapphub/hevm
-    $ cd hevm && cabal configure && cabal install
+If you can't or won't use Nix, the easiest way especially if you don't have GHC (the Haskell compiler) installed already, is to use [Stack](https://docs.haskellstack.org/en/stable/README/), which can take care of installing GHC for you.  These commands should work:
+
+```
+$ curl -sSL https://get.haskellstack.org/ | sh
+$ git clone https://github.com/dapphub/dapptools.git
+$ cd dapptools/src/hevm && stack setup && stack install
+```
+Also, hevm is in Hackage so you can execute `stack install hevm` to get it up and running.
+
+If you prefer to use your own installation of GHC and the basic Haskell package manager, Cabal, simply run:
+
+```
+$ git clone https://github.com/dapphub/dapptools.git
+$ cd dapptools/src/hevm && cabal configure && cabal install
+```
+
+**Note:** If you are on Mac OS environment when building with Stack you will have to previously deal with `readline` and `secp256k1`. These commands should be enough (rearding you have brew already installed):
```

Comment:
If you are on macOS

Reply:
> rearding you have brew already installed

What's `rearding` here?

Reply:
Typo, wanted to write `regarding` trying to say "I suppose you have already installed brew"